src/core/ext/transport/chttp2/transport/hpack_parser_table.h

```diff
@@ -126,11 +141,17 @@ class HPackTable {
     // Maximum number of entries we could possibly fit in the table, given
     // defined overheads.
     uint32_t max_entries_ = hpack_constants::kInitialTableEntries;
+    // Which index holds a timestamp (or kNoTimestamp if none do).
+    static constexpr uint32_t kNoTimestamp =
```

Comment:
the first time I read the comment, I thought that we are saving the timestamp in one of the indices, but we are really saving the timestamp associated with an index